collector/imagescollector.go

```diff
@@ -143,14 +143,17 @@ func getAllImageNames() ([]string, error) {
 		logrus.Warnf("Error while running docker image list : %s", err)
 		return nil, err
 	}
-	images := strings.Split(string(outputStr), "\n")
```

Comment:
There is no empty string check for `outputStr`. Do you want to add a check?

Reply:
Done.